File: apps/python/lib/domain/type/fizz_buzz_type_02.py
from lib.domain.model.fizz_buzz_value import FizzBuzzValue
from lib.domain.type.fizz_buzz_type import FizzBuzzType


class FizzBuzzType02(FizzBuzzType):
    """タイプ 2: 数をそのまま文字列で返す。"""

    def generate(self, number: int) -> FizzBuzzValue:
        return FizzBuzzValue(number, str(number))

File: apps/php/src/Domain/Type/FizzBuzzType02.php
<?php

declare(strict_types=1);

namespace App\Domain\Type;

use App\Domain\Model\FizzBuzzValue;

final class FizzBuzzType02 implements FizzBuzzType
{
    public function generate(int $number): FizzBuzzValue
    {
        return new FizzBuzzValue($number, (string) $number);
    }
}
